jsettlers.logic/src/jsettlers/logic/buildings/Building.java

```diff
@@ -88,7 +82,7 @@
 	private ShortPoint2D pos;
 	private IBuildingsGrid grid;
 	private Player player;
-	private byte state = STATE_CREATED;
+	private EBuildingState state = EBuildingState.CREATED;
```

Comment:
I dont understand why you are so careful not to break replays and savegames?! I usually play one scenario until I win or lose - and would never in between checkout the newest version from github, compile it and keep playing... I thought thats why there are official builds?!